packages/components/src/__overlays__/Tooltip/v3/ToggleTip/ToggleTip.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { HTMLAttributes, ReactNode, useContext, useRef } from "react"
+import classnames from "classnames"
+import { AriaButtonOptions, useTooltipTrigger } from "react-aria"
+import { TooltipTriggerStateContext } from "react-aria-components"
+import styles from "./ToggleTip.module.scss"
+
+export type ToggleTipProps = {
+  children: ReactNode
+  className?: HTMLAttributes<HTMLButtonElement>["className"]
+} & AriaButtonOptions<"button">
+
+export const ToggleTip = ({
+  children,
+  className,
+  ...props
+}: ToggleTipProps): JSX.Element => {
+  const ref = useRef<HTMLButtonElement>(null)
+  const state = useContext(TooltipTriggerStateContext)!
+
+  // registers ref to triggerRef for positioning the tooltip
+  // it also registers event handler for escape key to close the tooltip
+  // other handlers like onKeyDown or onPointerDown we provide own
+  useTooltipTrigger(props, state, ref)
```

Comment:
Where did we land on replacing this with Popover?

Reply:
We've landed that it's 2 similar yet different things. The popover will be likely styled differently, will be in general bigger & heavier and will have different focus considerations. So in general intended for larger amount of content.

Reply:
After more a11y complications that appeared yesterday we've decided to remove ToggleTip for now and keep it only as Popover. If there's significant demand in future we can rethink but for now it doesn't seems to be worth it